### ISSUE TITLE
Frequent crash in WindowChromeBehavior -> GetHitTestResult

### ISSUE THREAD

**Issue by Aaron-Stewart-RPS** [ControlzEx/ControlzEx#193]
## Describe the bug

I am getting a fairly frequent reports from the field as follows:
```
System.InvalidOperationException: This Visual is not connected to a PresentationSource.
- at System.Windows.Media.Visual.PointFromScreen(Point point)
- at ControlzEx.Behaviors.WindowChromeBehavior.GetHitTestResult(IntPtr lParam) in C:\projects\controlzex\src\ControlzEx\Behaviors\WindowChrome\WindowChromeBehavior.MessageHandling.cs:line 470
- at ControlzEx.Behaviors.WindowChromeBehavior.WindowProc(IntPtr hwnd, Int32 msg, UIntPtr wParam, IntPtr lParam, Boolean& handled) in C:\projects\controlzex\src\ControlzEx\Behaviors\WindowChrome\WindowChromeBehavior.MessageHandling.cs:line 139
- at ControlzEx.Behaviors.WindowChromeBehavior.WindowProc(IntPtr hwnd, Int32 msg, IntPtr wparam, IntPtr lparam, Boolean& handled) in C:\projects\controlzex\src\ControlzEx\Behaviors\WindowChrome\WindowChromeBehavior.MessageHandling.cs:line 112
- at System.Windows.Interop.HwndSource.PublicHooksFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
- at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
- at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
- at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
- at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

That is the simplest call stack I have but I have a few others that all point back to GetHitTestResult in WindowChromeBehavior.

## Steps to reproduce

Unfortunately I cannot reproduce it locally, all I have are application crash reports.

## Environment

```
ControlzEx version: v6.0.0
Windows build number: Microsoft Windows NT 10.0.14393.0
Target Framework: .Net Framework 4.0.30319.42000
```

**Comment by batzen** [comment]
That issue should already be fixed in the upcoming 7.0 version.

**Comment by Aaron-Stewart-RPS** [comment]
Oh that’s great – thank you for taking the time to let me know.

Aaron.
